src/components/camera/Client.ts

```diff
@@ -0,0 +1,79 @@
+import { type Camera, MimeType, Properties } from './Camera'
+import { CameraServiceClient } from '../../gen/component/camera/v1/camera_pb_service.esm'
+import type Client from '../../Client'
+import type { HttpBody } from '../../gen/google/api/httpbody_pb'
+import pb from '../../gen/component/camera/v1/camera_pb.esm'
+import { promisify } from '../../utils'
+
+export class CameraClient implements Camera {
+  private client: Client | undefined
+  private readonly name: string
+
+  constructor (client: Client, name: string) {
+    this.client = client
+    this.name = name
+  }
+
+  private get cameraService () {
+    if (!this.client) {
+      throw new Error('not connected yet')
+    }
+    const { grpcOptions, serviceHost } = this.client.serviceConnection
+    return new CameraServiceClient(serviceHost, grpcOptions)
+  }
+
+  async getImage (mimeType: MimeType): Promise<Uint8Array> {
```

Comment:
We should probably add tests for all of these methods using a mock camera. I think its fine that we dont have any in this pr since this is the first wrapper but they should be added in the future.

Reply:
For sure - I'm working on setting up unit tests and mocks in a separate branch: https://github.com/viamrobotics/viam-typescript-sdk/tree/RSDK-915-add-testing-to-ts-sdk-ci-pipeline.